shared/src/business/useCases/getNotificationsInteractor.ts

```diff
@@ -1,50 +1,80 @@
 import { CHIEF_JUDGE, ROLES } from '../entities/EntityConstants';
+import { isEmpty } from 'lodash';
+
+const setJudgeUser = async (
+  judgeUserId: string,
+  applicationContext: IApplicationContext,
+  currentUser: any,
+) => {
+  let judgeUser = null;
+
+  if (judgeUserId) {
+    judgeUser = await applicationContext
+      .getPersistenceGateway()
+      .getUserById({ applicationContext, userId: judgeUserId });
+  } else if (currentUser.role === ROLES.adc) {
+    judgeUser = {
+      name: CHIEF_JUDGE,
+    };
+  }
+  return judgeUser;
+};
 
 /**
  * getNotificationsInteractor
  *
  * @param {object} applicationContext the application context
  * @param {object} providers the providers object
+ * @param {object} providers.caseServicesSupervisorData optional caseServicesSupervisorData containing section
  * @param {object} providers.judgeUser optional judgeUser for additional filtering
  * @returns {object} inbox unread message counts for the individual and section inboxes
  */
 export const getNotificationsInteractor = async (
   applicationContext: IApplicationContext,
-  { judgeUserId }: { judgeUserId: string },
+  {
+    caseServicesSupervisorData,
+    judgeUserId,
+  }: { judgeUserId: string; caseServicesSupervisorData: any },
 ) => {
   const appContextUser = applicationContext.getCurrentUser();
+
   const currentUser = await applicationContext
     .getPersistenceGateway()
     .getUserById({ applicationContext, userId: appContextUser.userId });
 
-  let judgeUser = null;
+  const judgeUser = await setJudgeUser(
```

Comment:
nit: might rename to getJudgeUser since nothing is being set anywhere

Reply:
Good point! ☑️ on it